### ISSUE TITLE
[Feature] Display matched bookings on home page itself

### ISSUE THREAD

**Issue by sal2701** [divyamagwl/PoolIIIT#47]
If there is a booking with similar timings then display it on the homepage itself.

Also, a mail can be sent once a booking matches.

**Comment by jaggu21** [comment]
@sal2701 ,
The bookings with similar timings can be viewed in "My Bookings" page.The link to My bookings page is present in the home page itself.

![Image](https://user-images.githubusercontent.com/56889104/84662076-ed26e500-af38-11ea-87c0-fd25ac81eef0.png)

I was thinking of the mail feature as well.However,sending mail after every booking match can result in spamming.Consider a peak time , for example the last day of the semester ,there would be atleast 10 people travelling to the airport at the same time(or similar timings).This means the user would recieve around 15 mails.
 Instead I was thinking sending a mail to the user 2 or 3 days earlier to the scheduled date and let the user know if there are any matches with his booking.

**Comment by sal2701** [comment]
Yeah but it can be easier for the user if you display it on the homepage as well. Not the bookings necessarily. But a simple message that you have new matching bookings. A message beside the "My Bookings" button to let them know to go there. A small change but will be useful.

Regarding the mails, true there might be a spam. You should decide on a proper protocol and and give an option to unsubscribe, brainstorm and come up with something

**Comment by jaggu21** [comment]
> Yeah but it can be easier for the user if you display it on the homepage as well. Not the bookings necessarily. But a simple message that you have new matching bookings. A message beside the "My Bookings" button to let them know to go there. A small change but will be useful.
> 
> Regarding the mails, true there might be a spam. You should decide on a proper protocol and and give an option to unsubscribe, brainstorm and come up with something

Sounds good. We will work on it.
